demos/JWST/S1_nirx_template.ecf

```diff
@@ -22,6 +22,9 @@ skip_jump           False
 skip_ramp_fitting   False
 skip_gain_scale     False
 
+#Pipeline stages parameters
+jump_rejection_threshold  3.0 #float, default is 4.0, CR sigma rejection threshold
```

Comment:
Can this be added to the S1 MIRI template as well? Also, why set this to a value which is not the default in the template? Do you know/suspect that 3.0 is better? If not, it should be left at the default. Also, is it 4.0 for all of NIRCam, NIRSpec, and NIRISS by default? If not, we should allow None to be the default in Eureka and only update the `jwst` parameter if `meta.jump_rejection_threshold` is not None.

Reply:
Good point, it slipped my mind that MIRI has its own S1 file and I've changed it back to default 4.0 now.
Regarding `None`, I changed it so that it's default when `meta.jump_rejection_threshold` is not a float and save it in the log.